garage.com/src/test/java/ConvertibleCarTests.java

```diff
@@ -0,0 +1,53 @@
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class ConvertibleCarTests    {
+	private ConvertibleCar Car;
+
+	@BeforeClass
+	public void setup() {
+		String model = "Miata";
+		IEngine engine = new SmallEngine();
+		IIgnition ignitionSystem = new ElectronicIgnition();
+		
+		this.Car = new ConvertibleCar(model, engine, ignitionSystem);
+	}
+
+	@Test
+	public void canBuildConvertibleCar() {
+		ConvertibleCar car = this.Car;
+
+		String actualModel = car.getModel();
+
+		assertEquals(actualModel, Car.getModel() , "both should the same car model");
```

Comment:
This will always pass because you are comparing the same value. You set the actualModel on line 24.